ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -1503,17 +1327,6 @@ periodics:
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-arm-node-density-heavy-24nodes
-  reporter_config:
```

Comment:
looks like all the slack integrations were taken out... :(

Reply:
not sure why - but anytime I rebase against master these are missing :/